infra/src/rbac.tf

```
@@ -3,8 +3,19 @@ data "azurerm_data_factory" "adf" {
   resource_group_name = "io-p-rg-operations"
 }
 
-resource "azurerm_role_assignment" "adf_to_cosmos_account_reader" {
-  scope                = module.cosmosdb_account.id
-  role_definition_name = "Cosmos DB Account Reader Role"
-  principal_id         = data.azurerm_data_factory.adf.identity[0].principal_id
+module "adf_to_cosmos_data_reader_db" {
+  source  = "pagopa-dx/azure-role-assignments/azurerm"
+  version = "1.0.2"
```

```suggestion
  version = "~> 1.0"
```
To get patch and minor versions automatically!